test/Masa.Contrib.Data.IdGenerator.Snowflake.BenchmarkDotnet.Tests/DistributedBenchmarks.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the MIT License. See LICENSE.txt in the project root for license information.
+
+namespace Masa.Contrib.Data.IdGenerator.Snowflake.BenchmarkDotnet.Tests;
+
+[SimpleJob(RunStrategy.ColdStart, RuntimeMoniker.Net60, targetCount: 1000)]
```

Comment:
Add the benchmark result sample to the readme.

Reply:
processed